### PR TITLE
Update documentation 

### PR DESCRIPTION
**Pull request by lionel-nj**
**Summary:**

This PR updates the content of file README.md to emphasize the use of locally stored datasets.
This also provides documentation about the use of new command line option `exclude` with short name `-x`.

Please make sure these boxes are checked before submitting your pull request - thanks!

- ~[ ] Run the unit tests with `gradle test` to make sure you didn't break anything~
- ~[ ] Format the title like "Fix #<issue_number> - <short description of fix and changes>" (for example - "Fix #1111 - Check for null value before using field")~
- ~[ ] Linked all relevant issues~
- ~[ ] Include screenshot(s) showing how this pull request works and fixes the issue(s)~

### **Note: This PR takes changes provided in https://github.com/MobilityData/gtfs-validator/pull/278 , these should be merged before this documentation is updated.**

### CODE REVIEW

**Comment by lionel-nj** [comment]
Question: we would like to encourage our software users to validate locally stored GTFS datasets rather than validating GTFS datasets stored on `openmobilityda.org`. 

In this documentation, more precisely in the sample usage that we provide, should we totally disregard this last functionality and only provide sample usage to validate locally stored GTFS datasets? Or should we make this doc the most generic possible: provide 2 sample usages: 
- one explaining how to validate a locally stored dataset, 
- then another one detailing how to validate a GTFS dataset stored on `openmobilitydata.org`?

**Comment by fabrice-v** [comment]
Re: Question
I think the advisable thing would be to describe the url capability without pointing to `openmobilitydata.org`

File: README.md [hunk 1]
```diff
@@ -18,13 +18,13 @@ This command-line tool written in Java that performs the following steps:
...
 ``` 
-java -jar gtfs-validator-v1.1.0.jar -u https://transitfeeds.com/p/mbta/64/latest/download -z input.zip -e input -o output
+java -jar gtfs-validator-v1.1.0.jar -i relative_path_to_zipped_dataset -o relative_output_path -e relative_extraction_path.zip -x agency.txt,routes.txt
 ```
 
 ...which will:
- 1. Download the GTFS feed at the URL `https://transitfeeds.com/p/mbta/64/latest/download` and name it `input.zip`
- 1. Extract the `input.zip` contents to the directory `input`
- 1. Validate the GTFS data and output the results to the directory `output`. Validation results are exported to JSON by default.
+ 1. Search for a zipped GTFS dataset located at `relativr_path_to_zipped_dataset` and name it `relative_extraction_path.zip`
```
Comment (fabrice-v) [inline README.md:25]:
```suggestion
 1. Search for a zipped GTFS dataset located at `relative_path_to_zipped_dataset`
```

File: README.md [hunk 2]
```diff
@@ -18,13 +18,13 @@ This command-line tool written in Java that performs the following steps:
...
+ 1. Extract the `relative_extraction_path.zip` contents to the directory located at `relative_extraction_path`
```
Comment (fabrice-v) [inline README.md:26]:
```suggestion
 1. Extract the zip content to a directory located at `relative_extraction_path`
```

File: README.md [hunk 3]
```diff
@@ -18,13 +18,13 @@ This command-line tool written in Java that performs the following steps:
...
+ 1. Validate the GTFS data and output the results to the directory located at `relative_output_path`. Validation results are exported to JSON by default. The validation process will not be executed on the enumeration of files provided via option `-x` and the files that rely on them.
```
Comment (fabrice-v) [inline README.md:28]: Add 1 or more examples detailing what folders would be created and their content as well as how the `-x` option works

**Pull request merged**